src/impl_al.hpp

```diff
@@ -46,6 +47,7 @@ inline void onError(Error error) {
 inline bool alCheck() {
 	if (auto err = alGetError(); err != AL_NO_ERROR) {
 		onError(Error::eOpenALError);
+		std::cerr << err << std::endl;
```

Comment:
Why another `cerr`? `onError` is the customization point that `cerr`s by default; this way users have no way to suppress library output, and it occurs twice by default.

Reply:
Oops, that was left from a debugging session, removing later